packages/client/src/publication/graphql/publication.graphql

```diff
@@ -236,6 +236,30 @@ query PublicationMetadataStatus($request: GetPublicationMetadataStatusRequest!)
   }
 }
 
+query GetProfileBookmarks(
```

Comment:
```suggestion
query PublicationsProfileBookmarks(
```
in client sdk I'm trying to keep it as close to the API as possible for consistency